isis/src/control/apps/findfeatures/findfeatures.xml

```diff
@@ -1543,6 +1589,33 @@ End
           <filter>*.conf</filter>
       </parameter>
 
+      <parameter name= "GLOBALS">
+            <type>string</type>
+            <brief>String containing global parameters and values</brief>
+            <description>
+                <p>
+                    This string can contain additional parameters that will
+                    set or reset global parameters provded by other mechanisms.
+                    This program option is primarily useful for making small
+                    scale adjustments to algorithm parameters in a convenient
+                    and efficient manner.
+                </p>
+                <p>
+                    For example, using this parameter is the most straightforward
+                    way to select the FASTGEOM algorithm. To select the FastGeom
+                    Grid algorithm and constrain the number of interations to 10,
+                    use GLOBALS=FastGeomAlgorithm:Grid@FastGeomGridStopIteration:10".
+                </p>
+                <p>
+                    There is very little robust error detection or any confirmation
+                    that the parameter names are valid. Any misspelled parameters
+                    are not detected and ill-formed parameter strings may not
+                    result in errors, but passthrough and mangle other parameter
+                    construction.
+                </p>
+            </description>
+          <internalDefault>None</internalDefault>
+      </parameter>
```

Comment:
This seems extremely specific to power users of find features. Without any kind of error checking or other mechanisms it seems like people could really shoot themselves in the foot here.

Plus what variables does this expose? If I dig into the findfeatures code I can find them but they aren't exposed in a way that would even indicate they are present. As a user on the surface I would have no idea how this worked or what it's for.

It seems like this is handled by the `Parameters` input in the application as per the tests, but from the documentation it would seem like the fastgeom parameters should not be part of the `Parameters` file.

I am sure there are a couple ways this could be addressed, but I think adding a `fastgeomparamters` file option if `FastGeom` is enabled would be nice. Either that or this `Global` parameter should only be for `FastGeom` parameters.

More than happy to iterate on a solution here

Reply:
Maybe something to consider in the future is reworking findfeatures parameter tuning to be a little more consistent. It seems like one could make any change they want from the `GLOBAL` parameter and again end up doing unintended things by overriding variables in a passed in parameter file. For new users it's going to be overwhelming if they have to understand PVL parameter files as well as passing in variables from the `GLOBAL` parameter. At which point they would probably just pick whatever path gives them the least resistance.

Some potential in the future would be having different fields for different parameter sets, getting away from parameter files and moving more towards this key, value text parsing that the `GLOBAL` parameter is doing.

I think adding some context for `FASTGEOM` into the XML for the app somewhere would be ideal, basically context around the FASTGEOM parameters and describing the `GLOBAL` parameter. Seems like a straight forward place for that information to live. 

The other additions also sound useful and would get this closed out!

Reply:
The revisions I pushed hopefully resolves these issues/concerns.

Please review and let me know if there is anything remaining.